Jenkinsfile

```diff
@@ -1,5 +1,17 @@
 #!groovy
 
+properties(
+    [
+        parameters(
+            [
+                booleanParam(name: 'ARCHIVE', defaultValue: false, description: 'Archive the build artifacts by pushing to an S3 bucket.'),
+                string(name: 'CONTAINERD_REF', defaultValue: 'master', description: 'Git ref of containerd repo to build.'),
+                string(name: 'RUNC_REF', defaultValue: '', description: 'Git ref of runc repo to build.'),
```

Comment:
Will this cause the default "automatic selection from upstream" to be skipped? https://github.com/docker/containerd-packaging/blob/484fe7cdeaeb2711cb0a823bff4d5ee51009f0ff/common/common.mk#L27